jerry-core/ecma/operations/ecma-promise-object.c

```diff
@@ -437,6 +468,28 @@ ecma_promise_create_resolving_functions (ecma_object_t *object_p, /**< the promi
   /* 5. - 7. */
   funcs->reject = ecma_promise_create_resolving_functions_helper (object_p,
                                                                   ecma_promise_reject_handler);
+  if (!create_already_resolved)
+  {
+    return;
+  }
+
+  ecma_value_t already_resolved = ecma_op_create_boolean_object (ECMA_VALUE_FALSE);
+  ecma_string_t *str_already_resolved_p = ecma_get_magic_string (LIT_INTERNAL_MAGIC_STRING_ALREADY_RESOLVED);
+  ecma_property_value_t *value_p;
+
+  value_p = ecma_create_named_data_property (ecma_get_object_from_value (funcs->resolve),
+                                             str_already_resolved_p,
+                                             ECMA_PROPERTY_FIXED,
+                                             NULL);
+  value_p->value = already_resolved;
+
+  value_p = ecma_create_named_data_property (ecma_get_object_from_value (funcs->reject),
```

Comment:
Is it possible to have the same kind of test for "reject" handler like the "resolve" one has?

Reply:
What do you mean?

Reply:
A test case just like the one added in this PR. But probably reject behaves a bit differently.

Reply:
Ok, I get it now. Reject cannot be "delayed", that is a special feature of "resolve". Probably it would be too easy to start an infinite reject loop :)